test-suite/js/scalajs-test-suite-2.10-fastopt.html

```diff
@@ -1,17 +1,16 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Scala.js Test Suite - Jasmine HTML reporter</title>
+  <title>Scala.js Test Suite</title>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-  <link rel="stylesheet" type="text/css" href="http://cdn.jsdelivr.net/jasmine/1.3.1/jasmine.css"></link>
+  <link rel="stylesheet" type="text/css" href="test-stylesheet.css">
+  <script type="text/javascript">
+    var __ScalaJSEnv = {"javaSystemProperties": { "scalajs.fastopt-stage": "true" }};
```

Comment:
Currently I set these in the runner itself, but it is arguably nicer if I do it here.